flink-cdc-migration-tests/pom.xml

```diff
@@ -19,13 +19,10 @@ limitations under the License.
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <parent>
```

Comment:
Actually you can keep the <parent> section here. Removing `flink-cdc-migration-tests` from the <modules> section in the parent pom should be enough.